### ISSUE TITLE
 feature request: custom start and end scramble/timer

### ISSUE THREAD

**Issue by TheValll** [cs0x7f/cstimer#426]
Get the possibility at the user to custom his scramble/timer between 2 steps.
We can already start the scramble/timer at a custom cfpo step for example.

Here we get a scramble who start at the oll and the timer stop until the end of the resolution :

![image](https://github.com/user-attachments/assets/d141add0-8359-44db-ab89-571e9329f18c)

I suggest here to get the possibility to stop the timer when we want.

Here an example of the feature :
![image](https://github.com/user-attachments/assets/c4e1fbc6-2c71-4e57-87ed-8143d0e4e348)

The user can start his scramble with an easy x-cross and end the timer when the oll is complete.

**Comment by cs0x7f** [comment]
Do you mean that when using bluetooth cubes, the timer might be stopped when some phases are completed?

**Comment by TheValll** [comment]
Yes I didn't precise, I think it's useless when you don't use bluetooth cube cause you always need to let go your cube for touch a key.
